personal_tokens.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.3
+info:
+  title: Personal Tokens - OpenAPI
+  description: |
+    The Personal Tokens API allows you to manage your existing personal tokens.
+  license:
+    name: MIT
+    url: https://github.com/aziontech/azionapi-openapi/blob/main/LICENSE.md
+  version: 1.0.0
+externalDocs:
+  description: See the documentation on Personal Tokens for more information.
+  url: https://www.azion.com/en/documentation/products/accounts/personal-tokens/
+servers:
+  - description: Production
+    url: https://api.azionapi.net
+security:
+  - tokenAuth: []
+tags:
+  - name: Personal Token
+    description: List all personal tokens
+paths:
+  /iam/personal_tokens:
+    get:
+      tags:
+        - Personal Token
+      summary: List all existing personal token 
+      description: List all existing personal token
+      operationId: ListPersonalToken
+      
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonalTokenResponseWithResults'          
+components:
+  schemas:
+    PersonalTokenResponseWithResults:
+      type: object
+      properties:
+        count:
+          type: integer
+          format: int64
+          example: 2
```

Comment:
Remove all examples. In this case, you are providing an integer as an example of an integer. It's so obvious that we shouldn't consume file space with it.

Reply:
done!https://github.com/aziontech/azionapi-openapi/pull/65/commits/10f48c580e73f9dc2a6a71cdb25d371166cfa7ee